packages/wrangler/src/render-helpers.ts

```diff
@@ -0,0 +1,32 @@
+import EventEmitter from "node:events";
```

Comment:
I don't think we should use this in the actual app, only as a test helper. Because when we do want to add functionality interactivity to a component we're rendering, it'll get hard to unwind this usage from tests and such.

Reply:
We can already use the `ink-testing-library` for testing static Ink components directly right now.
What we can't do is test such components within the context of their usage.

If we don't use this in the app, then there is no need for it.